azuread/resource_service_principal_test.go

```diff
@@ -131,7 +133,8 @@ resource "azuread_application" "test" {
 }
 
 resource "azuread_service_principal" "test" {
-  application_id = "${azuread_application.test.application_id}"
+	application_id = "${azuread_application.test.application_id}"
+	app_role_assignment_required = true
```

Comment:
COuld we align these assingments here?

Reply:
Done

Reply:
`oauth2_permissions` would be great, tags however we always leave at the end. Thanks!